packages/bits/src/lib/select-v2/option-key-control.service.ts

```diff
@@ -111,4 +117,18 @@ export class OptionKeyControlService<T extends IOption> {
             });
         }
     }
+
+    private announceNavigatedOption(): void {
+        const activeItem = this.keyboardEventsManager.activeItem;
+
+        if (activeItem) {
+            this.liveAnnouncer.announce((activeItem as any).value);
+        }
+    }
+
+    private announceDropdown(open: boolean) {
+        const message = open ? ANNOUNCER_OPEN_MESSAGE(this.optionItems.length) : ANNOUNCER_CLOSE_MESSAGE;
```

Comment:
See my previous comment for context:
```suggestion
        const message = open ? `${this.optionItems.length} ${ANNOUNCER_OPEN_MESSAGE_SUFFIX}` : ANNOUNCER_CLOSE_MESSAGE;
```